service/src/main/java/bio/terra/pipelines/stairway/imputation/SubmitCromwellRunSetStep.java

```diff
@@ -233,11 +144,20 @@ public StepResult doStep(FlightContext flightContext)
                         new OutputDestinationRecordUpdate()
                             .recordAttribute("n_failed_chunks")
                             .type(OutputDestination.TypeEnum.RECORD_UPDATE)))
+            // define the WDS record to link to for inputs and outputs
             .wdsRecords(
                 new WdsRecordSet()
                     .recordType(pipelineName.getValue())
                     .addRecordIdsItem(flightContext.getFlightId()));
 
+    // add inputs
+    List<WorkflowInputDefinition> cbasWorkflowInputDefinitions =
+        pipelinesService.prepareCbasWorkflowInputRecordLookupDefinitions(
+            pipelinesService.getAllPipelineInputDefinitions(pipelineName), wdlMethodName);
```

Comment:
would it be better to put this into the inputParameters at the start of the flight, rather than making this call to the TSPS database now?

Reply:
we can either put it in the beginning of the flight or put it in the parameter map when we make the same call in `PrepareImputationInputsStep.java`.  I like the idea of doing it up front since every imputation pipeline will have inputs